bitnami/mariadb/templates/primary/statefulset.yaml

```diff
@@ -189,6 +189,12 @@ spec:
             {{- end }}
             - name: MARIADB_DATABASE
               value: {{ .Values.auth.database | quote }}
+            - name: MARIADB_ENABLE_SSL
+              value: {{ ternary "yes" "no" .Values.tls.enabled | quote }}
+            {{- if .Values.tls.enabled }}
+            - name: MYSQL_CLIENT_CA_FILE
+              value: {{ printf "/opt/bitnami/mariadb/ca/%s" .Values.tls.certCAFilename | quote }}
+            {{- end }}
```

Comment:
Is this mandatory? could be the ca optional? Some CertManager issuers don't include the CA in the secret.